rotkehlchen/db/cache.py

```diff
@@ -16,6 +17,22 @@ class DBCacheStatic(Enum):
     LAST_WITHDRAWALS_EXIT_QUERY_TS = 'last_withdrawals_exit_query_ts'
 
 
+class LocationLocationNameType(TypedDict):
+    """Type of kwargs, used to get the value of `DBCacheDynamic.LAST_CRYPTOTX_OFFSET`"""
+    location: str
+    location_name: str
+
+
+class LocationLocationNameAccountIdType(LocationLocationNameType):
+    """Type of kwargs, used to get the value of `DBCacheDynamic.LAST_QUERY_TS` and `DBCacheDynamic.LAST_QUERY_ID`"""  # noqa: E501
+    account_id: str
+
+
+class AddressType(TypedDict):
```

Comment:
Do you need a TypeDict for this? Since it's just one can't you just use kwargs: str